tnb/banks.py

```diff
@@ -2,42 +2,42 @@
 
 
 class Bank(BaseClient):
-    def fetch_accounts(self) -> object:
+    def fetch_accounts(self) -> list:
```

Comment:
Please don't take into account the return type used in the test. To know the return type of a function, look [here](http://143.110.137.54).
If you want to know the return type of `fetch_accounts`, click on `/accounts` and you'll see that this is a dict and not a list.

Reply:
@VincentHch No, these changes need to be implemented in the new tests PR https://github.com/thenewboston-developers/thenewboston-python-client/issues/46

Reply:
Ok I trust you, but it is not writen in the description of the issue therefore we should warn the assignee. He is not aware of that

Reply:
There is request for change from me on the PR.